deps/rabbit/priv/schema/rabbit.schema

```diff
@@ -2483,6 +2483,8 @@ end}.
 {mapping, "quorum_queue.compute_checksums", "rabbit.quorum_compute_checksums", [
     {datatype, {enum, [true, false]}}]}.
 
+{mapping, "quorum_queue.property_equivalence.skip_checks_on_redeclaration", "rabbit.quorum_skip_checks_on_redeclaration", [
```

Comment:
Since we ended up performing some checks for common optional queue arguments,
this settings should be renamed to `quorum_queue.property_equivalence.relaxed_checks_on_redeclaration` or something like that.